tests/keyword_test.fbs

```diff
@@ -0,0 +1,14 @@
+
+enum ABC: int { void, where, stackalloc }
```

Comment:
I'm not sure that we'd want to add tests for these, since a) since they only test a few keywords the value of this is low, and b) we'd need separate ones for every language which would be a lot of clutter. So I'd be fine leaving this out, wdyt @dbaileychess ?

Reply:
I'm happy to take this out if you feel they don't add value.  I used them while developing to make sure that the generated code was correct and I got everything right.

My only counter argument of why they are helpful is that's a pretty niche use case, and I would expect a lot of people developing on the code generator don't have examples of these in their code base and as a result, these could be easily overlooked during a code refactor.

So even though it's a small set of the keywords, I think just having anything in there is helpful to make sure that this continues to work going forward.  I don't think we would actually need new ones for each language, I think any language that supports keyword disambiguation should just add a few language specific keywords to keyword_test.fbs in the various enums, struct, tables and include their language in the generate_code.sh for this test.  Then their language will be protected going foward.

Reply:
Fwiw, I think its a good idea to have these. We also don't need a keyword_test.fbs for each language, but rather just bundle all keywords of all languages into this file.